warning C4800: 'uint32_t': forcing value to bool 'true' or 'false' (performance warning)

Have been seeing the following warning for the past few releases (including `1.39.1`) when building on Windows. Here is the [attached build log]( https://github.com/WebAssembly/binaryen/files/3834232/windows_binaryen_azure_build_log.txt ) and a snippet of the warning seen below. This can also be seen in [this Azure build]( https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=92148&view=logs ).

```C++
d:\bld\binaryen_1573453236089\work\src\wasm-features.h(71): warning C4800: 'uint32_t': forcing value to bool 'true' or 'false' (performance warning)
d:\bld\binaryen_1573453236089\work\src\wasm-features.h(72): warning C4800: 'uint32_t': forcing value to bool 'true' or 'false' (performance warning)
d:\bld\binaryen_1573453236089\work\src\wasm-features.h(73): warning C4800: 'uint32_t': forcing value to bool 'true' or 'false' (performance warning)
d:\bld\binaryen_1573453236089\work\src\wasm-features.h(74): warning C4800: 'uint32_t': forcing value to bool 'true' or 'false' (performance warning)
d:\bld\binaryen_1573453236089\work\src\wasm-features.h(75): warning C4800: 'uint32_t': forcing value to bool 'true' or 'false' (performance warning)
d:\bld\binaryen_1573453236089\work\src\wasm-features.h(76): warning C4800: 'uint32_t': forcing value to bool 'true' or 'false' (performance warning)
d:\bld\binaryen_1573453236089\work\src\wasm-features.h(77): warning C4800: 'uint32_t': forcing value to bool 'true' or 'false' (performance warning)
d:\bld\binaryen_1573453236089\work\src\wasm-features.h(78): warning C4800: 'uint32_t': forcing value to bool 'true' or 'false' (performance warning)
d:\bld\binaryen_1573453236089\work\src\wasm-features.h(79): warning C4800: 'uint32_t': forcing value to bool 'true' or 'false' (performance warning)
```

--- a/src/wasm-features.h
+++ b/src/wasm-features.h
@@ -73,7 +73,9 @@ struct FeatureSet {
   bool hasSIMD() const { return bool(features & SIMD); }
   bool hasBulkMemory() const { return bool(features & BulkMemory); }
   bool hasSignExt() const { return bool(features & SignExt); }
-  bool hasExceptionHandling() const { return bool(features & ExceptionHandling); }
+  bool hasExceptionHandling() const {
+    return bool(features & ExceptionHandling);
+  }
   bool hasTailCall() const { return bool(features & TailCall); }
   bool hasReferenceTypes() const { return bool(features & ReferenceTypes); }
   bool hasAll() const { return bool(features & All); }

--- a/src/wasm-features.h
+++ b/src/wasm-features.h
@@ -67,16 +67,16 @@ struct FeatureSet {
 
   bool isMVP() const { return features == MVP; }
   bool has(Feature f) { return (features & f) == f; }
-  bool hasAtomics() const { return features & Atomics; }
-  bool hasMutableGlobals() const { return features & MutableGlobals; }
-  bool hasTruncSat() const { return features & TruncSat; }
-  bool hasSIMD() const { return features & SIMD; }
-  bool hasBulkMemory() const { return features & BulkMemory; }
-  bool hasSignExt() const { return features & SignExt; }
-  bool hasExceptionHandling() const { return features & ExceptionHandling; }
-  bool hasTailCall() const { return features & TailCall; }
-  bool hasReferenceTypes() const { return features & ReferenceTypes; }
-  bool hasAll() const { return features & All; }
+  bool hasAtomics() const { return bool(features & Atomics); }
+  bool hasMutableGlobals() const { return bool(features & MutableGlobals); }
+  bool hasTruncSat() const { return bool(features & TruncSat); }
+  bool hasSIMD() const { return bool(features & SIMD); }
+  bool hasBulkMemory() const { return bool(features & BulkMemory); }
+  bool hasSignExt() const { return bool(features & SignExt); }
+  bool hasExceptionHandling() const { return bool(features & ExceptionHandling); }
+  bool hasTailCall() const { return bool(features & TailCall); }
+  bool hasReferenceTypes() const { return bool(features & ReferenceTypes); }
+  bool hasAll() const { return bool(features & All); }
 
   void makeMVP() { features = MVP; }
   void set(Feature f, bool v = true) {

